Clang with -Wstrict-prototypes fails to build

```
FAILED: PureSpice/CMakeFiles/purespice.dir/src/agent.c.o
/usr/lib/ccache/bin/clang  -I/var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999 -I/var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999/client/include -I/var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999_build -I/var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999_build/include -I/var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999/repos/nanosvg/src -I/var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999/repos/PureSpice/include -I/var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999/repos/PureSpice/src -I/usr/include/spice-1  -march=native -O3 -pipe -flto=thin -fstack-protector-strong -D_FORTIFY_SOURCE=2   -D ENABLE_EGL -D ATOMIC_LOCKING -D GL_GLEXT_PROTOTYPES -D USE_NETTLE -march=x86-64-v2 -Wall -Wextra -Wno-sign-compare -Wno-unused-parameter -Wstrict-prototypes -Werror -Wfatal-errors -ffast-math -fdata-sections -ffunction-sections -std=gnu11 -MD -MT PureSpice/CMakeFiles/purespice.dir/src/agent.c.o -MF PureSpice/CMakeFiles/purespice.dir/src/agent.c.o.d -o PureSpice/CMakeFiles/purespice.dir/src/agent.c.o -c /var/tmp/portage/app-emulation/looking-glass-9999/work/looking-glass-9999/repos/PureSpice/src/agent.c
../looking-glass-9999/repos/PureSpice/src/agent.c:513:32: fatal error: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
bool purespice_clipboardRelease()
                               ^
                                void
```

--- a/src/agent.c
+++ b/src/agent.c
@@ -510,7 +510,7 @@ bool purespice_clipboardGrab(PSDataType types[], int count)
   return true;
 }
 
-bool purespice_clipboardRelease()
+bool purespice_clipboardRelease(void)
 {
   if (!agent.present)
     return false;

--- a/src/ps.c
+++ b/src/ps.c
@@ -284,7 +284,7 @@ err_host:
   return false;
 }
 
-void purespice_disconnect()
+void purespice_disconnect(void)
 {
   if (!g_ps.initialized)
   {

